packages/react-core/src/components/Card/__tests__/CardHeader.test.tsx

```diff
@@ -13,6 +15,16 @@ describe('CardHeader', () => {
     const { asFragment } = render(<CardHeader actions="test" />);
     expect(asFragment()).toMatchSnapshot();
   });
+
+  test(`Should render CardHeader without ${styles.modifiers.wrap} by default`, () => {
+    const { asFragment } = render(<CardHeader data-testid="card-header" />);
+    expect(screen.getByTestId('card-header')).not.toHaveClass('pf-m-wrap');
```

Comment:
So the asFragment is a leftover that some tests have due to usnot getting to refactoring them yet. Ideally we want to avoid that when adding new tests. All that needs to be done for these 2 new tests is update the `render` lines more to match the tests you had added to  your Toolbar PR.

Also can we update the `pm-m-wrap` on line 21 to use the styles object as well